chapter-09/end/Objects/BulletSprite.cs

```diff
@@ -0,0 +1,22 @@
+﻿using chapter_09.Engine.Objects;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+
+namespace chapter_09.Objects
+{
+    public class BulletSprite : BaseGameObject
+    {
+        private const float BULLET_SPEED = 10.0f;
+
+        public BulletSprite(Texture2D texture)
+        {
+            _texture = texture;
+            AddBoundingBox(new Engine.Objects.BoundingBox(new Vector2(9, 4), 10, 22));
```

Comment:
Is the texture these dimensions so we could just pull from the Texture2D object instead of the magic numbers?

Reply:
no, the texture is bigger than the bullet itself, unfortunately. I hate those magic numbers though. There everywhere in this PR.